Dockerfile

```diff
@@ -0,0 +1,106 @@
+# Build:
+#   docker build -t discourse/ruby:3.3-bookworm-slim --build-arg DEBIAN_RELEASE=bookworm --build-arg RUBY_VERSION=3.3.3 .
+#
+# You are not expected to build and push this Docker image by hand. This is done by the `build` Github Actions workflow.
+ARG DEBIAN_RELEASE=
+ARG RUBY_VERSION=
+
+FROM debian:${DEBIAN_RELEASE}-slim
+
+ARG RUBY_VERSION
+
+ENV LANG en_US.UTF-8 \
+  LANGUAGE en_US.UTF-8 \
+  LC_ALL en_US.UTF-8 \
+  RUBY_VERSION=3.3.3
+
+# Installs system dependencies required to run Ruby
+RUN set -eux; \
```

Comment:
Note that this is one gigantic `RUN` command to keep the image as small as possible. If we split things up, the image will be larger and this is a known issue with Docker.

Reply:
btw, I recently saw this - https://github.com/discourse-org/docker-haproxy/blob/main/config/Dockerfile#L8

a neat way to avoid (some) line continuation characters 🙂